classes/PodsAdmin.php

```diff
@@ -2090,7 +2090,7 @@ public function admin_components () {
             if ( !empty( $component_data[ 'category' ] ) ) {
                 $category_url = pods_query_arg( array( 'view' => sanitize_title( $component_data[ 'category' ] ), 'pg' => '', 'page' => $_GET[ 'page' ] ) );
 
-                $component_data[ 'category' ] = '<a href="' . esc_url( $category_url ) . '">' . $component_data[ 'category' ] . '</a>';
+                $component_data[ 'category' ] = '<a href="' . esc_url( $category_url ) . '">' . __( $component_data[ 'category' ], 'pods' ) . '</a>';
```

Comment:
It's working for me, the categories appear translated because it already exists somewhere in the code.
Still, I dont like it much, it would be better allow the Name, Category and Description in the components headers correctly localizable with gettext functions.